docs/proposed/http-storage-node-protocol.rst

```diff
@@ -350,6 +350,9 @@ Because of the simple types used throughout
 and the equivalence described in `RFC 7049`_
 these examples should be representative regardless of which of these two encodings is chosen.
 
+For CBOR messages, any sequence that is semantically a set (i.e. no repeated values allowed, order doesn't matter, and is hashable in Python) should be sent as a set.
+Tag 6.258 is used to indicate sets in CBOR.
```

Comment:
Is this magic number our invention or does it derive from some specification?  If the latter a reference would be nice.